hal/Cargo.toml

```diff
@@ -163,3 +163,7 @@ samv71q20b = ["atsamv71q20b", "v71", "pins-144", "flash-1M", "device-selected"]
 samv71q20b-rt = ["samv71q20b", "atsamv71q20b/rt"]
 samv71q21b = ["atsamv71q21b", "v71", "pins-144", "flash-2M", "device-selected"]
 samv71q21b-rt = ["samv71q21b", "atsamv71q21b/rt"]
+
+[package.metadata.docs.rs]
+# enable all possible features for docs.rs
+features = ["samv71q21b-rt", "unproven", "usart-spi-host-without-select", "reconfigurable-system-pins"]
```

Comment:
This may be missing:
```toml
default-target = "thumbv7em-none-eabihf"
```
Otherwise building `cortex-m` dependencies is going to fail.

Reply:
Good catch, all set